aspnetcore/fundamentals/servers/kestrel/http3.md

```diff
@@ -49,12 +64,15 @@ HTTP/3 is configured on app start-up. The following code:
 * Sets `EnableAltSvc` to `true` on Kestrel options.
 * Configures port 5001 to use `HttpProtocols.Http1AndHttp2AndHttp3`.
 
-This sample code is specific to .Net 6 Preview 7, and will change in .Net 6 RC 1.
+This sample code is specific to .NET 6 Preview 7, and will change in .NET 6 RC 1.
 
 [!code-csharp[](samples/6.x/Http3Sample/Program.cs?name=snippet_UseHttp3&highlight=8)]
 
 For more information on building the host, see the **Set up a host** and **Default builder settings** sections of <xref:fundamentals/host/generic-host#set-up-a-host>.
 
 ## Localhost testing
+
 * Browsers do not enable HTTP/3 on localhost/loopback connections: to test with a browser, run the client and server on separate machines.
-* HttpClient can be used for localhost/loopback testing.
+* `HttpClient` can be used for localhost/loopback testing in .NET 6 or later. Extra configured is required when using `HttpClient` to make an HTTP/3 request:
+  * Set `HttpRequestMessage.Version` to 3.0, or
+  * Set `HttpRequestMessage.VersionPolicy` to `HttpVersionPolicy.RequestVersionOrHigher`.
```

Comment:
Is this a preview7 specific constraint because of alt-svc? It's working on rc1 without these.